frontend/src/components/Pages/FrontPage/RobotCards/PressureStatusView.tsx

```diff
@@ -14,12 +15,15 @@ const StyledTopography = styled(Typography)<{ $fontSize?: 24 | 16 | 18 | 32 | 40
 `
 export interface PressureStatusViewProps {
     pressureInBar?: number
+    pressureInMilliBar?: number
     itemSize?: 24 | 16 | 18 | 32 | 40 | 48 | undefined
     upperPressureWarningThreshold?: number
     lowerPressureWarningThreshold?: number
+    robotStatus?: RobotStatus
```

Comment:
Do we handle it if the robotStatus is not provided?

Reply:
Now we do. It is always mandatory and always sent when we choose to display pressure and battery props.